File: 241027/숫자 사이의 배수/multiplication-between-numbers.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cin >> a >> b;
    int sum_val = 0, count = 0;
    double avg;

    for (int i = a; i <= b; i++) {
        if (i % 5 == 0 || i % 7 == 0) {
            count++;
            sum_val += i;
        }
    }

    avg = (sum_val / count); 
    cout << sum_val << " " << avg;
    return 0;
}

File: 241027/숫자 사이의 배수/multiplication-between-numbers.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ");

let a = Number(input[0]);
let b = Number(input[1]);

let sumVal = 0, avg = 0, count = 0;

for (let i = a; i <= b; i++) {
    if (i % 5 === 0 || i % 7 === 0) {
        count++;
        sumVal += i;
    }
}

avg = (sumVal / count);

console.log(sumVal, parseFloat(avg).toFixed(1));